benchmark/code/minimatch.js

```diff
@@ -1,12 +1,11 @@
 'use strict';
 
-
 var minimatch = require('minimatch').match;
 var multimatch = require('multimatch');
 
 module.exports = function (files, patterns, options) {
   if (Array.isArray(patterns)) {
-    return multimatch(files, patterns, options);
+    return multimatch.apply(null, arguments);
```

Comment:
I still don't quite understand. How does the prior code cause the args to be passed incorrectly? Is there a larger indeterminate number of args that can be passed than those 3?